plugins/backend/catalog-backend-module-aws/src/providers/AWSDynamoDbTableProvider.ts

```diff
@@ -89,7 +89,6 @@ export class AWSDynamoDbTableProvider extends AWSEntityProvider {
                 spec: {
                   owner: this.accountId,
                   type: 'dynamo-db-table',
```

Comment:
Do we have a provider to get the rows and map them to a resource with a custom type?

Reply:
The DDB data provider allows to map rows into entities based on configuration values the user can define